<UI version="4.0" >
 <class>MyWidgetClass</class>
 <widget class="QWidget" name="MyWidgetClass" >
	 <property name="geometry">
		 <rect>
			 <x>0</x>
			 <y>0</y>
			 <width>800</width>
			 <height>600</height>
		 </rect>
	 </property>
	 <property name="windowTitle">
		 <string>MyWidget</string>
	 </property>
	 <widget class="QLabel" name="lblArray">
		 <property name="geometry">
			 <rect>
				 <x>70</x>
				 <y>20</y>
				 <width>231</width>
				 <height>471</height>
			 </rect>
		 </property>
		 <property name="font">
			 <font>
				 <pointsize>11</pointsize>
			 </font>
		 </property>
		 <property name="styleSheet">
			 <string notr="true">background-color: white;</string>
		 </property>
		 <property name="frameShape">
			 <enum>QFrame::Box</enum>
		 </property>
		 <property name="text">
			 <string/>
		 </property>
		 <property name="alignment">
			 <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
		 </property>
	 </widget>
	 <widget class="QPushButton" name="btnFindNum">
		 <property name="geometry">
			 <rect>
				 <x>410</x>
				 <y>270</y>
				 <width>191</width>
				 <height>51</height>
			 </rect>
		 </property>
		 <property name="font">
			 <font>
				 <pointsize>13</pointsize>
			 </font>
		 </property>
		 <property name="text">
			 <string>Найти в массиве</string>
		 </property>
	 </widget>
	 <widget class="QLineEdit" name="editNumToFind">
		 <property name="geometry">
			 <rect>
				 <x>410</x>
				 <y>210</y>
				 <width>191</width>
				 <height>31</height>
			 </rect>
		 </property>
	 </widget>
	 <widget class="QLabel" name="label_2">
		 <property name="geometry">
			 <rect>
				 <x>410</x>
				 <y>180</y>
				 <width>171</width>
				 <height>31</height>
			 </rect>
		 </property>
		 <property name="font">
			 <font>
				 <pointsize>11</pointsize>
			 </font>
		 </property>
		 <property name="text">
			 <string>Введите число</string>
		 </property>
		 <property name="buddy">
			 <cstring>editNumToFind</cstring>
		 </property>
	 </widget>
	 <widget class="QLabel" name="lblOutput">
		 <property name="geometry">
			 <rect>
				 <x>410</x>
				 <y>340</y>
				 <width>371</width>
				 <height>61</height>
			 </rect>
		 </property>
		 <property name="font">
			 <font>
				 <pointsize>11</pointsize>
			 </font>
		 </property>
		 <property name="styleSheet">
			 <string notr="true"/>
		 </property>
		 <property name="frameShape">
			 <enum>QFrame::NoFrame</enum>
		 </property>
		 <property name="text">
			 <string/>
		 </property>
		 <property name="alignment">
			 <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
		 </property>
	 </widget>
	 <widget class="QSpinBox" name="editArraySize">
		 <property name="geometry">
			 <rect>
				 <x>230</x>
				 <y>490</y>
				 <width>71</width>
				 <height>51</height>
			 </rect>
		 </property>
		 <property name="font">
			 <font>
				 <pointsize>12</pointsize>
			 </font>
		 </property>
	 </widget>
	 <widget class="QPushButton" name="btnGenerate">
		 <property name="geometry">
			 <rect>
				 <x>70</x>
				 <y>540</y>
				 <width>231</width>
				 <height>31</height>
			 </rect>
		 </property>
		 <property name="font">
			 <font>
				 <pointsize>10</pointsize>
			 </font>
		 </property>
		 <property name="text">
			 <string>Сгенерировать</string>
		 </property>
	 </widget>
	 <widget class="QLabel" name="label">
		 <property name="geometry">
			 <rect>
				 <x>90</x>
				 <y>500</y>
				 <width>121</width>
				 <height>21</height>
			 </rect>
		 </property>
		 <property name="font">
			 <font>
				 <pointsize>12</pointsize>
			 </font>
		 </property>
		 <property name="text">
			 <string>Размер массива:</string>
		 </property>
		 <property name="buddy">
			 <cstring>editArraySize</cstring>
		 </property>
	 </widget>
 </widget>
 <layoutDefault spacing="6" margin="11" />
 <pixmapfunction></pixmapfunction>
 <resources>
   <include location="mywidget.qrc"/>
 </resources>
 <connections/>
</UI>
